### ISSUE TITLE
Required PHP version doesn't match code

### ISSUE THREAD

**Issue by garrettw** [Level-2/Transphporm#156]
Your `composer.json` specifies a minimum PHP version of 5.5, however I've run into this:
`PHP Parse error:  syntax error, unexpected '?' in /src/Parser/Value.php on line 150`

It's not liking the null coalesce operator, which is, of course, only in 7.0+.
What is your preferred course? Change `composer.json` to require 7.0, or make the code compatible with 5.6?

**Comment by solleer** [comment]
See #152. I think @TomBZombie is going to change this to be php 5.6 compatible

**Comment by garrettw** [comment]
Oh, my bad. In that case maybe I'll submit a new PR to fix it once my other one is merged in.
